src/app/dashboard/dashboard.component.ts

```diff
@@ -18,10 +21,17 @@ export class DashboardComponent implements OnInit {
    * User  city name
    */
   city = '';
+  /**
+   * Forecast data
+   */
+  forecast: ForecastModel = new ForecastModel();
 
   isDataReady = false;
 
-  constructor(protected ipApiService: IpApiService) {}
+  constructor(
+    protected ipApiService: IpApiService,
+    protected owService: OpenWeatherService
+  ) {}
 
   async ngOnInit(): Promise<void> {
     this.updateData();
```

Comment:
we should start by initLocation and then set dataReady to true.
Please validate if this works